new evaluations.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+        <src >
+        <link rel="stylesheet"  href = "assets/css.css">
+        <div class="topnav" id="myTopnav">
```

Comment:
You've just uncovered one of my pet peeves: naming stuff by prefixing it with `my`. Why are you so possessive, my friend? :crying_cat_face: